web/api/v1/api.go

```diff
@@ -159,13 +159,13 @@ type apiFuncResult struct {
 
 type apiFunc func(r *http.Request) apiFuncResult
 
-// TSDBAdmin defines the tsdb interfaces used by the v1 API for admin operations.
-type TSDBAdmin interface {
+// TSDBAdmin defines the tsdb interfaces used by the v1 API for admin operations as well as statistics.
+type TSDBAdminStats interface {
```

Comment:
if that rename is really needed, comment should be updated too

Reply:
Yea, it's no longer just admin.

Reply:
Thanks!